doc/connectivity/bluetooth/bluetooth-le-host.rst

```diff
@@ -153,6 +153,117 @@ an application comes in direct contact with it only when using it in the
+      Retransmission mode and Flow Control Mode, not in Basic L2CAP mode.
+
+  * - Protocol Data Unit (PDU)
+    - Packet of data containing L2CAP data. PDUs always start with Basic L2CAP
+      header.
+
+      Types of PDUs for LE: :ref:`B-frames <bluetooth_l2cap_b_frame>` and
+      :ref:`K-frames <bluetooth_l2cap_k_frame>`.
+
+      Types of PDUs for BR/EDR: I-frames, S-frames, C-frames and G-frames.
+
+  * - Maximum Transmission Unit (MTU)
+    - Maximum size of an SDU that the upper layer is capable of accepting.
+
+  * - Maximum Payload Size (MPS)
+    - Maximum payload size that the L2CAP layer is capable of accepting.
+
+      Without segmentation or in Basic L2CAP mode, the MTU size is the same as
+      the MPS.
+
+  * - Basic L2CAP header
+    - Present at the beginning of each PDU. It contains two fields, the PDU
+      length and the Channel Identifier (CID).
+
+PDU Types
+---------
+
+.. _bluetooth_l2cap_b_frame:
+
+B-frame: Basic information frame
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+PDU used in Basic L2CAP mode. It contains a complete SDU as its payload.
+
+.. image:: img/l2cap_b_frame.png
+  :align: center
+  :width: 45%
+  :alt: Representation of a B-frame PDU. The PDU is split into two rectangles,
+        the first one being the L2CAP header, its size is 4 octets and its made
+        of the PDU length and the channel ID. The second rectangle represent the
+        information payload and its size is less or equal to MPS.
+
+.. _bluetooth_l2cap_k_frame:
+
+K-frame: Credit-based frame
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+PDU used in LE Credit Based Flow Control mode and Enhanced Credit Based Flow
+Control mode. It contain a SDU segment and additional protocol information.
+
+.. image:: img/l2cap_k_frame_1.png
+  :width: 45%
+  :alt: Representation of a starting K-frame PDU. The PDU is split into three
+        rectangles, the first one being the L2CAP header, its size is 4 octets
+        and its made of the PDU length and the channel ID. The second rectangle
+        represent the L2CAP SDU length, its size is 2 octets. The third
+        rectangle represent the information payload and its size is less or
+        equal to MPS. The information payload contain the L2CAP SDU.
+
+.. image:: img/l2cap_k_frame.png
+  :align: right
+  :width: 45%
+  :alt: Representation of K-frames PDUs after the starting one. The PDU is split
+        into two rectangles, the first one being the L2CAP header, its size is 4
+        octets and its made of the PDU length and the channel ID. The second
+        rectangle represent the information payload and its size is less or
+        equal to MPS. The information payload contain the L2CAP SDU.
+
+Relevant Kconfig
+----------------
+
+.. list-table::
+  :header-rows: 1
+
+  * - Kconfig symbol
+    - Description
+
+  * - :kconfig:option:`CONFIG_BT_BUF_ACL_RX_SIZE`
+    - Represent the MPS
+
+  * - :kconfig:option:`CONFIG_BT_L2CAP_TX_MTU`
+    - Represent the L2CAP MTU, which is the :code:`ATT_MTU`
```

Comment:
The value of ATT_MTU is not in-general equal to any instance of a L2CAP MTU.

Reply:
In UATT it is, but yes this isn't true in EATT. I'll rephrase that

Reply:
I think it sounds wrong for UATT too. I just want to make sure our users understand that  ATT_MTU of a bearer is the minimum of the local and remote L2CAP MTU.

Reply:
maybe not mention ATT at all. You can make another doc like this one for ATT, and then make a link to the mtu part from here.